lib/parser.js

```diff
@@ -98,9 +100,18 @@ function _mapSymbol(symbol){
     symbol.type = types[0].types.join(' | ');
   }
 
-  if(returns.length === 1){
-    symbol["return"] = returns[0].types.join(' ');
-    symbol["returnDescription"] = returns[0].description.replace(/(<([^>]+)>)/ig, '');
```

Comment:
Indeed! Could you please update the template as well (removing returnDescription for instance)